dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -12,6 +12,27 @@ def wait_with_short_timeout
   wait_with_timeout(SHORT_WAIT_TIMEOUT)
 end
 
+def element_stale?(element)
+  begin
+    element.enabled?
+    false
+  rescue Selenium::WebDriver::Error::StaleElementReferenceError
+    true
+  end
+end
+
+def page_load(page_load)
```

Comment:
Nit: can we name the argument something more descriptive?  E.g. `should_wait_for_unload`.